backend/service/service_account.go

```diff
@@ -84,3 +100,87 @@ func isProfiler(address string) bool {
 	}
 	return false
 }
+
+func (service *AccountService) QueryDelegations(address string) (result []*vo.AccountDelegationsVo) {
+	delegations := lcd.GetDelegationsByDelAddr(address)
+	result = make([]*vo.AccountDelegationsVo, 0, len(delegations))
+	for _, val := range delegations {
+		data := vo.AccountDelegationsVo{
+			Address: val.ValidatorAddr,
+			Shares:  val.Shares,
+			Height:  val.Height,
+		}
+		valdator, err := document.Validator{}.QueryValidatorDetailByOperatorAddr(val.ValidatorAddr)
```

Comment:
don't execute db query in a loop, you can query validators by operator address list

Reply:
have done